app/policies/project_policy.rb

```diff
@@ -148,7 +148,7 @@ def export_sample_data?
   end
 
   def submit_workflow?
-    return true if Member.can_submit_workflow?(user, record) == true
+    return true if Member.can_submit_workflow?(user, record.namespace) == true
```

Comment:
**question:** Why isn't this just `return true if Member.can_submit_workflow?(user, record.namespace)`